contrib/storage-hbase/src/main/java/org/apache/drill/exec/store/hbase/config/HBasePersistentStore.java

```diff
@@ -148,13 +154,13 @@ private void delete(byte[] row) {
     private Result current = null;
     private Result last = null;
     private boolean done = false;
-    private int rowsRead = 0;
 
-    Iter() {
+    Iter(int take) {
       try {
         Scan scan = new Scan(tableNameStartKey, tableNameStopKey);
         scan.addColumn(FAMILY, QUALIFIER);
-        scan.setCaching(config.getMaxIteratorSize() > 100 ? 100 : config.getMaxIteratorSize());
+        scan.setCaching(Math.min(take, 100));
```

Comment:
nope. this patch introduces a way to paginate results via #getRange. this option is no longer needed.